templates/mcq.jsx

```diff
@@ -0,0 +1,104 @@
+import Adapt from 'core/js/adapt';
+import { templates, classes, html, compile } from 'core/js/reactHelpers';
+
+export default function(model, view) {
+  const data = model.toJSON();
+  const ariaLabels = Adapt.course.get('_globals')._accessibility._ariaLabels;
+
+  const isInteractive = model.isInteractive();
+
+  return (
+    <div className='component__inner mcq__inner'>
+
+      {templates.component(model, view)}
+
+      <div
+        className={classes([
+          'component__widget',
+          'mcq__widget',
+          !data._isEnabled && 'is-disabled',
+          data._isInteractionComplete && 'is-complete is-submitted show-user-answer',
+          data._isCorrect && 'is-correct'
+        ])}
+        role={data._isRadio ? 'radiogroup' : 'group'}
+      >
+
+        {data._items.map(({ text, _index, _isActive, _shouldBeSelected, _isHighlighted }, index) =>
+
+          <div
+            className={classes([
+              `mcq__item item-${index}`,
+              (!isInteractive && data._canShowMarking && _shouldBeSelected) ? 'is-correct' : null,
```

Comment:
Could this do with another variable?
```js
const shouldShowMarking = !isInteractive && data._canShowMarking;
```
The `classes()` function already handles falsies so `null` fallbacks aren't required:
```suggestion
              shouldShowMarking && _shouldBeSelected && 'is-correct',
```